bin/zeppelin-daemon.sh

```diff
@@ -190,6 +190,7 @@ function start() {
   wait_zeppelin_is_up_for_ci
   sleep 2
   check_if_process_is_alive
+  echo "ZEPPELIN_CLASSPATH: ${ZEPPELIN_CLASSPATH}" >> "${ZEPPELIN_LOGFILE}"
```

Comment:
Sure.

But one quick question, Do we really set or use ```ZEPPELIN_CLASSPATH_OVERRIDES``` ? I don't see any assignment operation on that property. Please let me know if  i am missing something.